### ISSUE TITLE
Possibly merge with Emblem.js repo?

### ISSUE THREAD

**Issue by machty** [patientslikeme/emblem2hbs#1]
Could you take a look at [this comment](https://github.com/machty/emblem.js/pull/188#issuecomment-70002462) in the Emblem HTMLbars PR? I'm considering making Emblem just output Hbs text, and some of the code in this repo might be useful towards that endeavor, so I'd be curious for your input.

**Comment by nbudin** [comment]
Thanks!  I definitely agree with you that emitting Handlebars text would be more maintainable going forward.  However, I'm not sure whether the code in emblem2hbs is useful for that or not.  What we're basically doing is:

1. running Emblem files through Emblem itself
2. looping through the emitted Handlebars AST and emitting Handlebars text
3. re-indenting the result (because Emblem produces conjoined, unformatted HTML for adjacent tags)

That process definitely works, but still relies on Handlebars AST as an intermediate.  I'm not sure if it's workable to produce Handlebars 1.x AST, do some stuff, and then run it through Handlebars 2.x or above in the same process.  (Although I'm also not sure it isn't!)

It seems like a cleaner pipeline would be to go straight from Emblem text to Handlebars text, if such a thing were possible.  But that is probably a lot more work.

In conclusion, I honestly don't know.  Regardless, if you'd like to have emblem2hbs in the emblem.js repo, I'm sure we can do that.
